src/vinylcontrol/vinylcontrolxwax.cpp

```diff
@@ -880,7 +880,7 @@ void VinylControlXwax::establishQuality(double& pitch) {
             static_cast<float>(std::accumulate(
                     m_iQualityRing, m_iQualityRing + m_iQualityRingFilled, 0)) /
             2.0 / 100.0 /
-            m_iQualityRingFilled; // Two information in percent per datapoint
+            static_cast<float>(m_iQualityRingFilled); // Two information in percent per datapoint
```

Comment:
the constants should be float as well.

Reply:
This is strange, I fixed his already in 6011087, but GitHub still showed the unfixed version to you. Please have a look at the latest commit.